src/RelayServer.ts

```diff
@@ -168,6 +172,23 @@ export class RelayServer extends EventEmitter {
     super();
     this.config = getServerConfig();
     this._provider = getProvider();
+    setProvider(this._provider);
+    const {
+      contracts: {
+        relayHubAddress,
+        deployVerifierAddress,
+        relayVerifierAddress,
+        smartWalletFactoryAddress,
+      },
+    } = this.config;
+    setEnvelopingConfig({
+      chainId: 33,
+      preferredRelays: [],
+      relayHubAddress,
+      deployVerifierAddress,
+      relayVerifierAddress,
+      smartWalletFactoryAddress,
```

Comment:
The RelayServer needs to use the `gasEstimator` and the `gasEstimator`, use the following functions from the client: `getSmartWalletAddress`,`estimateTokenTransferGas`,`estimateInternalCallGas`. Therefore the client needs to be configured before using it.